pom.xml

```diff
@@ -67,21 +63,21 @@
   </scm>
 
   <properties>
-    <jenkins.version>1.609.3</jenkins.version>
-    <java.level>7</java.level> <!-- sshd-core is 7+ -->
+    <jenkins.version>2.7.3</jenkins.version>
```

Comment:
@stephenc I consider bumping it even more to 2.73 in order to pick SSHD Core 1.6.0 from the core instead of bundling the current 1.0.0. WDYT?

Reply:
And removing `pluginFirstClassLoader`. Maybe, but a far riskier change, better left for another day.